server/lib/contributors.ts

```diff
@@ -220,63 +222,12 @@ const filterContributors = (contributors: ContributorsList, filters: Contributor
 // ---- Public API ----
 
 /**
- * Returns all the contributors for given collective
+ * Returns all the contributors for given collective.
+ * Prefer using the req.loaders.Contributors.forCollectiveId() from GraphQL resolvers.
  */
-export const getContributorsForCollective = async (
-  collectiveId: number,
-  filters: ContributorsFilters | null,
-): Promise<ContributorsList> => {
-  const contributorsCache: ContributorsCacheEntry = await loadContributors(collectiveId);
-  const contributors = contributorsCache.all || [];
-  return filterContributors(contributors, filters);
-};
-
-/**
- * Returns all the contributors for given collective
- */
-export const getPaginatedContributorsForCollective = async (
-  collectiveId: number,
-  filters: ContributorsFilters | null,
-): Promise<{
-  offset: number;
-  limit: number;
-  totalCount: number;
-  nodes: ContributorsList;
-}> => {
-  const contributorsCache: ContributorsCacheEntry = await loadContributors(collectiveId);
-  const contributors = contributorsCache.all || [];
-  const filteredContributors = filterContributors(contributors, omit(filters, ['offset', 'limit']));
-  return {
-    offset: filters?.offset || 0,
-    limit: filters?.limit || 0,
-    totalCount: filteredContributors.length,
-    nodes: !filters ? filteredContributors : filteredContributors.slice(filters.offset || 0, filters.limit),
-  };
-};
-
-/**
- * Returns all the contributors for given tier
- */
-export const getContributorsForTier = async (
-  collectiveId: number,
-  tierId: number,
-  filters: ContributorsFilters | null,
-): Promise<ContributorsList> => {
-  const contributorsCache: ContributorsCacheEntry = await loadContributors(collectiveId);
-  const contributors = contributorsCache.tiers[tierId.toString()] || [];
-  return filterContributors(contributors, filters);
-};
-
-/**
- * Get all the contributors that are not part of any tier.
- */
-export const getContributorsWithoutTier = async (
```

Comment:
This one was probably implemented for the "Custom contributions" card, but we're using the default `contributors` query with a filter on Tier in the frontend. Removing for now.